functions/src/controllers/locations.js

```diff
@@ -0,0 +1,116 @@
+const {
+    CREATED, getStatusText, INTERNAL_SERVER_ERROR, NOT_FOUND, OK,
+} = require('http-status-codes');
+const { db } = require('../utils/firebase');
+const errorHandler = require('../utils/errorHandler');
+
+const Locations = {
+    /**
+	 * The Location Routes
+	 * @function
+	 * @param {object} req - request object
+	 * @param {object} res - response object
+	 * @return  {Object} result
+	*/
+    async create(req, res) {
+        try {
+            const {
+                address, coords, country, image, lga, name, price, state, status, trafficRate,
+            } = req.body;
+            const data = {
+                address, coords, country, createdAt: new Date().toISOString(), image, lga,
```

Comment:
either destructure on the same line or on individual lines. Why is your eslint not catching?

Reply:
Please make sure that you have the eslint lint rules from the `project-template` installed in this repo

Reply:
I am trying to avoid the 25 function max line rule